docs/users-guide/interactive-dashboards.md

```diff
@@ -0,0 +1,92 @@
+## Interactive dashboards
+
+You can customize what happens when people click on questions in your dashboard. You can set up a question to:
+
+- send the user to a custom destination: a dashboard, question, or custom URL 
+- update a dashboard filter (what we call **cross-filtering**)
+
+To configure this interactivity, you'll use the **click behavior** option on a dashboard card to slide out the click behavior sidebar.
+
+### Click behavior sidebar
+
+From your dashboard, click on the **pencil** icon to enter dashboard edit mode. Hover over the card containing the question you want to customize. Metabase will display a menu at the top right of the card. Select the **Click behavior** option (the icon with the mouse cursor clicking on a card). 
+
+![Click behavior icon](images/interactive-dashboards/click-behavior-icon.png)
+
+Metabase will slide out the **Click behavior sidebar**. For questions composed using the query builder, you can select from 3 options:
+
+- Open the Metabase action menu.
+- Go to a custom destination.
+- Update a dashboard filter (if the dashboard has a filter)
+
+SQL questions will only have the option to **Go to a custom destination**, and **Update a dashboard filter**, as the action menu (which allows users to drill through the data) is only available to questions composed with the query builder.
+
+If your dashboard has a filter, you'll also see an option to update the filter (see [cross-filtering](#cross---filtering) below).
+
+![Click behavior menu](images/interactive-dashboards/click-behavior-menu.png)
+
+### Open the action menu
+
+For questions composed using the query builder, the default click behavior is to open the **action menu**, which presents people with the option to [drill through the data](https://www.metabase.com/blog/drilling-through-data/index.html):
+
+![Action menu](images/interactive-dashboards/action-menu.png)
+
+### Custom destinations
+
+You can set custom destinations for all questions, including SQL questions. 
+
+![Custom destinations](images/interactive-dashboards/custom-destination.png)
+
+Possible destinations include:
+
+- Dashboards
+- Saved questions
+- URLs
+
+You can also pass a value to that destination from the clicked card. For example, if you link to a dashboard that has a filter for category, you can pass a value for category from the origin question to the destination dashboard:
+
+![Pass value to dashboard](images/interactive-dashboards/pass-value.png)
+
+Once you select a column, Metabase will display the column used to pass the value, as well as the filter the value will be passed to at the destination dashboard.
```

Comment:
We should also find a way to work in the fact that you can also send the currently selected value of a dashboard filter on the current dashboard. And if you're using EE, you can pass a user attribute provided by SSO to the destination, too. Those values will show up as options when you click on one of the destination's filters, provided the values are compatible with that filter.